gui-doc/mrlib/scribblings/hierlist/hierlist.scrbl

```diff
@@ -9,6 +9,59 @@ A @racket[hierarchical-list%] control is a list of items, some of
 which can themselves be hierarchical lists. Each such sub-list has an
 arrow that the user can click to hide or show the sub-list's items.
 
+A short example to demonstrate this control:
+
+@racketblock[
```

Comment:
I didn't realize this was a `racket/gui` program at first. Maybe worth adding a note or `#lang` line.

(I know `codeblock` works for adding `#lang`:

```
@codeblock{
#lang racket/gui
(require mrlib/hierlist)
....
;; new class uses ....
....
}
```
)